include/aws/testing/aws_test_harness.h

```diff
@@ -62,9 +67,24 @@ static int s_cunit_failure_message0(
     s_cunit_failure_message0(FAIL_PREFIX, func, file, line, format, #__VA_ARGS__)
 
 static int total_failures;
+static int total_skip;
 
 #define SUCCESS (0)
 #define FAILURE (-1)
+#define SKIP (1)
```

Comment:
The skip code returned to cmake has to be in range [0,255]. Refer to cmake doc: https://cmake.org/cmake/help/latest/prop_test/SKIP_RETURN_CODE.html